packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-testing.md

```diff
@@ -2,104 +2,118 @@
-  - <label><input type="checkbox" /> When you navigate to an item by keyboard or by screen reader, you can easily understand and use the item. </label><br/>
-  - <label><input type="checkbox" /> The flow of information makes sense when navigating. Screen readers navigate the page in DOM order. If you used CSS to visually reposition elements, they might be announced in a nonsensical sequence. Instead, if you need something to appear earlier in the page, try to physically move it earlier in the DOM.</label>
+    <Checkbox label={<span> <b>Landmark</b> regions should have labels when there is more than one type of landmark, and they are not the same (such as navigation, main, form, etc.). Section element shouldn’t be used unless it has a label.</span>} id="general-criteria-8f" />
 
-**Accessibility points** <br />
-<label><input type="checkbox" /> **Structure:** The visual information architecture should map to the various rotor menus that exist by default. </label><br />
-- <label><input type="checkbox" /> **Check rotor:** Are there good headings, landmarks, links, form controls, and other elements? Do they make sense, and are they descriptive without overloading information? </label><br />
-  - <label><input type="checkbox" /> Headings: Check that heading levels convey structure/content and do not skip. A common practice is to use a single h1 for the primary headline or logo on a page, h2s for designating major sections, and h3s for supporting sections. </label><br />
-  - <label><input type="checkbox" /> Links </label><br />
-  - <label><input type="checkbox" /> Form controls </label><br />
-  - <label><input type="checkbox" /> Landmarks </label><br />
-  - <label><input type="checkbox" /> Tables </label><br />
+    <Checkbox label={<span><b>Tables</b> and table content is clearly described. WebAIM has additional guidance on <a href="https://webaim.org/techniques/tables/">creating accessible tables</a>.</span>} id="general-criteria-8g" />
 
+    <Checkbox label={<span><b>ARIA</b> labels provide descriptive details to screen reader users, without reiterating or overriding the text that is already there. If there is visible text then there doesn’t need to be an ARIA label. </span>} 
+    description={ <span>
+      <li> To add a non-visual label to an element you should use <span className="ws-code">aria-label</span>. Keep in mind you should not add this to an element that already has a visual label in the UI. You can also use <span className="ws-code">aria-labelledby</span> to label an element through existing text or <span className="ws-code">aria-label</span>. </li>
+      <li> <span> It’s important to understand that ARIA can only affect the semantics of an element; not the behavior. For example, while you can hide an element from screen readers with <span className="ws-code">aria-hidden=”true”</span>, that does not change the focus behavior for that element.</span></li>
```

Comment:
Not sure if this really fits here anymore since it's more about labels than general ARIA. @jessiehuff wdyt about whether this should stay here, removed, moved to the Product Dev Guide page, or a secret 4th option that I haven't found yet?

Reply:
Yeah, I see your point @thatblindgeye. Maybe it's best to just remove those subpoints? It looks like the the point about aria only affecting the semantics is already in the dev guide page, but do we have a section where we clarify the difference between `aria-label` and `aria-labelledby`? I wonder if it would make sense to remove that point here but just make sure that we have it clear in the dev docs somewhere?

Reply:
@thatblindgeye @jessiehuff I can see how the aria-labelledby point may be more helpful/visible in the dev guide. I went ahead and removed the subpoints here - I can add the aria-labelledby in a separate issue (or slip it into this PR if that seems okay) - wdyt?